diff -Nru PasteScript-1.7.3.orig/paste/script/templates.py PasteScript-1.7.3/paste/script/templates.py
--- PasteScript-1.7.3.orig/paste/script/templates.py	2009-03-29 19:07:49.323215602 +0200
+++ PasteScript-1.7.3/paste/script/templates.py	2009-03-29 19:15:58.876497792 +0200
@@ -49,7 +49,7 @@
         assert self._template_dir is not None, (
             "Template %r didn't set _template_dir" % self)
         if isinstance( self._template_dir, tuple):
-            return self._template_dir
+            return os.path.join(self.module_dir(), self._template_dir[1])
         else:
             return os.path.join(self.module_dir(), self._template_dir)
 
